packages/clerk-js/src/ui/elements/PhoneInput/useFormattedPhoneNumber.ts

```diff
@@ -5,7 +5,7 @@ import { extractDigits, formatPhoneNumber } from '../../utils';
 import type { CountryIso } from './countryCodeData';
 import { IsoToCountryMap } from './countryCodeData';
 
-type UseFormattedPhoneNumberProps = { defaultPhone?: string };
+type UseFormattedPhoneNumberProps = { defaultPhone?: string; defaultSelectedISO?: CountryIso };
```

Comment:
We use `ISO` instead of  `Iso` as we do everywhere else?  Lets still with camelCase for abbreviations :)

Reply:
@nikosdouvlis ok I'm switching from `defaultSelectedISO` to `defaultSelectedIso `